src/applications/disability-benefits/996/sass/0996-higher-level-review.scss

```diff
@@ -238,6 +238,14 @@ article[data-location="request-informal-conference"] {
   }
 }
 
+/* Step 4 Review Application */
+article[data-location="review-and-submit"] {
+  /* checkbox group "Required" lable (added to */
+  #root_scheduleTimes-label {
+    display: none;
+  }
+}
+
```

Comment:
Instead of adding all this would it be possible to add the class `.vads-u-display--none` to the item with id `root_scheduleTimes-label`?  [More info here](https://design.va.gov/utilities/display)

Reply:
That bit of css is actually hiding a form added `(*Required)` label that is added to a required [checkbox group](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/forms/available-features-and-usage-guidelines/#checkbox-group), but the design doesn't want to show the label but still show the error

Design

![](https://user-images.githubusercontent.com/136959/69181633-0d8ede00-0ad5-11ea-83eb-f53783ae188c.png)

Note: The bolded first paragraph is inconsistent with the implementation, and is being discussed with design

Implementation (without the above css added)

![](https://user-images.githubusercontent.com/136959/69182131-05836e00-0ad6-11ea-9890-92e3767498a9.png)

I did try adding a `widgetClassNames` ui option, but it gets applied to the individual checkboxes based on the schema I shared above.